hr_hr.lang

```diff
@@ -1,215 +1,349 @@
+gui.xaero_teleport_default_command=Uobičajena naredba teleportacije
+gui.xaero_waypoints_distance_min=Min. udaljenost prikaza PT
+gui.xaero_other_teams=Drugi timovi
+gui.xaero_entity_info_armour_numbers=Oklop u brojevima
+gui.xaero_entity_info_armour=Vrijednost oklopa
+gui.xaero_arrow_scale=Skala strelice
+gui.xaero_arrow_colour=Boja strelice
+gui.xaero_smooth_dots=Glatke točke
+gui.xaero_cave_zoom=Zoom-in u špiljama
+gui.xaero_lock_player_heads=Zaključaj glave igrača
+gui.xaero_safe_mode_box=Koristi Java (CPU) baziran ekvivalent moda umjesto OpenGL (GPU). Sigurni način je plan B, u slučaju da mod normalno ne uspije. Ne rade sve značajke u sigurnom načinu rada.
+gui.xaero_height_limit=Visinski limit entiteta
+gui.xaero_world_map=Karti svijeta
+gui.xaero_use_world_map=Koristi dijelove svijeta karte svijeta
+gui.xaero_patron_capes=Patreon plaštevi
+gui.xaero_entity_radar=Radar entiteta
+gui.xaero_entity_info_potion_effects=Efekti napitaka (samo u SP)
+gui.xaero_entity_info_potion_effects_scale=Skala efekata napitaka
+gui.xaero_terrain_depth=Dubina terena
+gui.xaero_terrain_slopes=Padine terena
+gui.xaero_block_transparency=Transparentnost blokova
+gui.xaero_team=Tim
+gui.xaero_world_server=Svijet/Poslužitelj
+gui.xaero_subworld_dimension=Podsvijet/Dimenzija
+gui.xaero_switch_waypoint_set=Zamjeni set putnih točaka
+gui.xaero_waypoint_opacity_ingame=Neprozirnost putne točke u igri
+gui.xaero_waypoint_opacity_map=Neprozirnost putne točke na karti
+gui.xaero_hide_world_names=Sakrij imena svijetova/IP-eve
+gui.xaero_open_slime=Otvori na pritisak tipke
+gui.xaero_options=Opcije
+gui.xaero_make_automatic=Napravi svijet/poslužitelj automatski
+gui.xaero_make_automatic_msg1=Jesi li siguran?
+gui.xaero_make_automatic_msg2=Ovo će zamijeniti sve podatke o putnim točkama za odabrani svijet/poslužitelj i automatski svijet/poslužitelj, na taj način čineći odabrani svijet automatskim. Budite sigurni da znate što radite.
+gui.xaero_make_multi_automatic=Napravi podsvijet automatski
+gui.xaero_make_multi_automatic_msg1=Jesi li siguran?
+gui.xaero_make_multi_automatic_msg2=Ovo će zamijeniti sve podatke o putnim točkama za odabrani podsvijet i automatski podsvijet, na taj način čineći odabrani podsvijet automatskim. Budite sigurni da znate što radite.
+gui.xaero_delete_world=Obriši svijet/poslužitelj
+gui.xaero_delete_world_msg1=Bi li htijeo obrisati sve podatke putnih točaka za odabrani svijet/poslužitelj?
+gui.xaero_delete_world_msg2=Ova akcija ne može biti vraćena. Budite sigurni da znate što radite.
+gui.xaero_delete_multi_world=Obriši podsvijet
+gui.xaero_delete_multi_world_msg1=Bi li htijeo obrisati sve podatke putnih točaka za odabrani podsvijet?
+gui.xaero_delete_multi_world_msg2=Ova akcija ne može biti vraćena. Budite sigurni da znate što radite.
+gui.xaero_always_show_distance=Uvijek prikazuj udaljenost od PT.
+gui.xaero_item_tooltip=Opis predmeta
+gui.xaero_entity_tooltip_settings=Postavke opisa predmeta
+gui.xaero_item_tooltip_min_lines=Minimum linija

```

Comment:
Your original text is "Minimum Lines", and my translation is correct, but if it were to be  "minimum number of text rows", I would put "minimalni broj redova teksta", but because it isn't, this is ok as it is now.

Reply:
But does "linija" mean a text row too?